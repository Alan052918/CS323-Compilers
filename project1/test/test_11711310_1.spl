int foo()
{
  int i = @;
  int i = ;
}

