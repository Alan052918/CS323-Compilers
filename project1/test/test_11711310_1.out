Error type A at Line 3: unknown lexeme @
Error type B at Line 4: Missing expression at the end of declaration
